configure

```diff
@@ -456,7 +456,7 @@ while true; do
   fi
 
   if is_linux; then
-    CUDNN_PATH_FROM_LDCONFIG="$(ldconfig -p | sed -n 's/.*libcudnn.so .* => \(.*\)/\1/p')"
+    CUDNN_PATH_FROM_LDCONFIG="$(/sbin/ldconfig -p | sed -n 's/.*libcudnn.so .* => \(.*\)/\1/p')"
```

Comment:
I feel like we need to be compatible with the previous case, where ldconfig is not under `/sbin`, but is in `PATH`.
For example, I have a centos system which is happy the way things are right now, but will break with this change as ldconfig resides in `/usr/sbin/ldconfig`
Can we modify these lines to say:
```
if ! type ldconfig > /dev/null; then
  LDCONFIG_BIN=/sbin/ldconfig
else
  LDCONFIG_BIN=ldconfig
fi
CUDNN_PATH_FROM_LDCONFIG="$($LDCONFIG_BIN -p | ......
```